### PR TITLE
V1.0

### PR DESCRIPTION
**Pull request by ThiagoHBA**

### CODE REVIEW

File: clio-app/Resource/en.lproj/ThemePhrases.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        "French Revolution: Goodbye monarchy, hello freedom!"
+    ],
+    "Geography": [
+        "Mount Everest: Earth's ultimate challenge for climbers.",
+        "Amazon Rainforest: The lungs of the Earth, wildlife's favorite spot.",
+        "Sahara Desert: Nature's colossal sandbox.",
+        "Great Barrier Reef: Nemo's underwater paradise.",
+        "Niagara Falls: Nature's water spectacle."
+    ],
+    "Philosophy": [
```
Comment (lvasantos) [inline clio-app/Resource/en.lproj/ThemePhrases.json]: Philosophy should be excluded.

File: clio-app/Resource/pt-BR.lproj/ThemePhrases.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        "Revolução Francesa: Adeus à monarquia, olá à liberdade!"
+    ],
+    "Geografia": [
+        "Monte Everest: O desafio supremo da Terra para alpinistas.",
+        "Floresta Amazônica: Os pulmões da Terra, o local favorito da vida selvagem.",
+        "Deserto do Saara: A caixa de areia colossal da natureza.",
+        "Grande Barreira de Coral: O paraíso submarino de Nemo.",
+        "Cataratas do Niágara: O espetáculo de águas da natureza."
+    ],
+    "Filosofia": [
```
Comment (lvasantos) [inline clio-app/Resource/pt-BR.lproj/ThemePhrases.json]: Filosofia should be excluded

**Review by lvasantos — CHANGES_REQUESTED** [review]
The philosophy prompts from JSON will not go to V1.0

**Review by BeatrizLdS — APPROVED** [review]
LGTM!

**Review by lvasantos — APPROVED** [review]

**Pull request merged**